mdn_theme/mdn/layout.html

```diff
@@ -1,56 +1,55 @@
-<!DOCTYPE html><html class="redesign no-js" dir="ltr" lang="en-US"><head prefix="og: http://ogp.me/ns#">
+<html data-ffo-opensanslight="false" data-ffo-opensans="false" lang="None" dir="ltr" class="redesign no-js"><head>
   <meta charset="utf-8">
   <meta content="IE=Edge" http-equiv="X-UA-Compatible">
   <script>(function(d) { d.className = d.className.replace(/\bno-js/, ''); })(document.documentElement);</script>
-    <title>{{ docstitle|e }} &amp;mdash; MDN</title>
+  
+  <title>{{ docstitle }} - MDN</title>
 
   <meta content="width=device-width, initial-scale=1" name="viewport">
   <meta content="index, follow" name="robots">
-  <link href="https://developer.mozilla.org/" rel="home"></link>
-  <link href="https://developer.mozilla.org/#copyright" rel="copyright"></link>
 
-      <link href="https://developer.mozilla.org/static/styles/font-awesome.css" media="all" rel="stylesheet"></link>
-<link href="https://developer.mozilla.org/static/styles/main.css" media="all" rel="stylesheet"></link>
+  
+    <link type="text/css" href="https://developer.mozilla.org/static/build/styles/mdn.65be1fdadb40.css" rel="stylesheet"></link>
```

Comment:
We keep asset files forever by default, like [fellowship.js](https://developer.cdn.mozilla.net/static/js/fellowship.js) from October 2015.  So, worst case is really old assets.